src/_docker_manager.sh

```diff
@@ -6,7 +6,7 @@ postgresComposeService="postgres"
 
 function compose_client() {
   flags=($@)
-  compose_files=$(for i in `ls ${PLEXTRAC_HOME}/docker-compose*.yml`; do printf " -f %s" "$i"; done )
+  compose_files=$(for i in `ls -r ${PLEXTRAC_HOME}/docker-compose*.yml`; do printf " -f %s" "$i"; done )
```

Comment:
I added this as an incidental - the order matters, and these were being applied in literally the _reverse_ I intended 😂 didn't cause an issue that I noticed until I tried to override the image for plextracdb and it didn't take.